mcp/include/star_camera_structs.h

```diff
@@ -0,0 +1,159 @@
+/* star_camera_structs.h: TIM data structures for the star cameras
+ *
+ * This software is copyright (C) 2021-2023 University of Arizona
+ *
+ * This file is part of the BLAST flight code licensed under the GNU
+ * General Public License.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software; if not, write to the Free Software Foundation,
+ * Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+// Defines for the star camera communications
+// on FC
+#define SC1_RECEIVE_PARAM_PORT "4970"
+#define SC2_RECEIVE_PARAM_PORT "4971"
+#define SC1_RECEIVE_SOLVE_PORT "4960"
+#define SC2_RECEIVE_SOLVE_PORT "4961"
+// on SCC
+#define SC1_COMMAND_PORT_FC1 "4950"
+#define SC2_COMMAND_PORT_FC1 "4950"
+#define SC1_COMMAND_PORT_FC2 "4951"
+#define SC2_COMMAND_PORT_FC2 "4951"
+
+// IP addresses for the SCs
+#define SC1_IP_ADDR "192.168.1.137"
+#define SC2_IP_ADDR "192.168.1.138"
+
+// Structure of the expected data packet to come down with every solution
+struct mcp_astrometry {
+    double ra_j2000;
+    double dec_j2000;
+    double ra_observed;
+    double dec_observed;
+    double rawtime;
+    double image_rms;
+    double fr;
+    double ps;
+    double ir;
+    double alt;
+    double az;
+    double photo_time;
+};
+
+// Command packet structure for the star cameras
+struct star_cam_capture {
```

Comment:
Update struct comments to be descriptive

Reply:
we'll see how round 2 looks.

Reply:
beaut